msf/palette.go

```diff
@@ -33,6 +33,7 @@ type Palette struct {
 	Model          Model            `json:"model"`
 	MaterialMeta   []Material       `json:"materialMeta"`
 	SpliceSettings []SpliceSettings `json:"spliceSettings"`
+	TreatAsSingleMaterial bool      `json:"treatAsSingleMaterial"`
```

Comment:
Should run `go fmt` (or configure your IDE to do this on save), the types should be vertically aligned